handler.go

```diff
@@ -67,10 +70,16 @@ func (h *Handler) Handle(res *Response) error {
 		return h.Run(res)
 	// if we don't find a match, return
 	default:
-		return nil
+		return ErrNotMatched
 	}
 }
 
+// IsRespond func
```

Comment:
Are these kind of redundant comments normal in Go? Or this is disabling some warnings or similar?

Reply:
They warn if an exported function or variable (those that starts with capital letter) is not commented. I missed a comment here.

![](https://media.giphy.com/media/Ob7p7lDT99cd2/giphy.gif)

Reply:
Fixed, thanks for pointing it out.